.github/workflows/ci.yaml

```diff
@@ -59,15 +58,13 @@ jobs:
 
   build:
     strategy:
-      fail-fast: false
```

Comment:
Any comments in the PR description why the fails-fast is no longer necessary? LGTM if expected to be removed. CC: @MiaAltieri